### PR TITLE
Move wiki in src

### PR DESCRIPTION
**Pull request by lorenaqendro**
### Description
Moving the wiki to the main code helps aligning it with versions of the code and also create pull requests for changes.

This PR contains only the logistic aspect of it (no changes to the content) and making sure that the links work properly. When reviewing pay special attention to the latter.

### Checklist
- [ ] Created new test/s (if possible)
- [x] Updated the README / documentation, if necessary

### CODE REVIEW

File: documentation/Recipe-Language.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+# Recipe language
+
+The core means of using the Tombolo Digital Connector is to create a data export recipes, describing the data or models that we want to get out of the system, without describing how to generate the data. The recipe language is expressed in the well-known JSON format.
+
+## Data export recipe
+A data export recipe consists of two parts:
+
+* **Dataset**: is a description of the data to be exported. The format of the dataset recipe is explained in the [Dataset recipe](#dataset-recipe) section below.
```
Comment (eddiejaoude) [inline documentation/Recipe-Language.md]: I would be consistent with the **markdown** style used.

For example here `**` is used here but in other places `__` is used:

- **Bold using \*\*** (`**Bold using asterix**`)
- __Bold using \_\___ (`__Bold using underscore__`)

File: documentation/Local-Datastore.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* ___subject:___ a foreign key reference to a Subject.
+* ___attribute:___ a foreign key reference to an Attribute.
+* _value:_ the actual value of the attribute for the subject, most often a string.
+
+## Timed Value
+__Timed Value__ is a data object representing the value of an attribute for a certain subject, taken at a certain time point.
+
+A timed value consists of:
+
+* ___subject:___ a foreign key reference to a Subject.
```
Comment (eddiejaoude) [inline documentation/Local-Datastore.md]: Different style for bold, mentioned [below](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/pull/377/files#r155160821)

File: documentation/Recipe-Language.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+## Field recipe
+The field recipe contains a list of fields that you want to output. Fields can be as simple as the latest value for a certain attribute, such as population density, or a potentially complex model, such as one for estimating the risk of social isolation among the elderly. All fields have two default fields (sic):
+
+* **fieldClass**: is the Java class used to calculate the field value
+* **label**: the label that the recipe creator chooses to associate with the field
+
+In addition each field has additional fields (sic) depending on the type of the field. For example a latest-value-field you need to specify the attribute for which you want the latest value. E.g.,
+
+```
```
Comment (theodosiakefala) [inline documentation/Recipe-Language.md]: This is missing the `json` tag.

File: documentation/Tutorial.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+with the cycling and air quality indicators for each of the 33 London boroughs. Opening the Json file in QGIS and using a quantile-based colouring of the boroughs should give you an image similar to the one below ([See QGIS tutorial for reference](http://www.qgistutorials.com/en/docs/basic_vector_styling.html)).
+
+![London borough cycle to car count ratio](https://user-images.githubusercontent.com/14051876/33561213-f9071faa-d909-11e7-98df-a8edae0c3a6a.png) 
+
+# Change granularity
+Suppose you want to get the cycling information at a finer granularity, say at [LSOA](Glossary#lsoa) level. Copy the data export recipe into a new file called `london-cycle-traffic-air-quality-lsoa.json`.
+
+Change the subjects clause from outputting all 33 London boroughs to outputting all LSOA geographies that fall inside the 33 London boroughs. I.e., change:
+
+```JSON
```
Comment (theodosiakefala) [inline documentation/Tutorial.md]: `json` tag should be in lowercase.

File: documentation/css/pdf.css [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    color: #0257c5;
+}
+
+em, i {
+    color: #006dfe;
+}
+
+b, strong {
+    color: #ff15a3;
+}
```
Comment (theodosiakefala) [inline documentation/css/pdf.css]: This is missing an empty line at the end of the file.

File: documentation/Tutorial.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+],
+```
+
+and add the following datasource to the list of datasources:
+
+```json
+{
+  "importerClass" : "uk.org.tombolo.importer.ons.OaImporter",
+  "datasourceId" : "lsoa"
+},
```
Comment (brettminnie) [inline documentation/Tutorial.md]: This trailing comma would make the json structure invalid. Please remove it.

File: documentation/Using-the-Digital-Connector.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+# Using the Digital Connector
+
+In a nutshell, using the Digital Connector involves four steps:
+
+ 1. After ensuring that you have installed the pre-requisites outlined [here](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector#quick-start), install the Digital Connector by cloning the GitHub repository.
+ 1. Write a recipe file that describes the desired data output from the Digital Connector. This recipe includes which data-sources to use and how to mix the data together.
```
Comment (brettminnie) [inline documentation/Using-the-Digital-Connector.md]: A link to a commented boilerplate recipe here would be very useful here.

**Comment by eddiejaoude** [comment]
@lorenaqendro @arya-hemanshu documentation will never be perfect, but I suggest to get this merge ASAP, because this will allow other people to submit PRs with improvements 🤓

**Comment by lorenaqendro** [comment]
@arya-hemanshu can you review this please?

**Pull request merged**